core/scripts/vrfv2/testnet/docker/wait-for-others/docker-wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
+# From https://github.com/vishnubob/wait-for-it
```

Comment:
is it ok to use this file in our Chainlink repo? It refers to a utility script which uses MIT license